test/fonts/test_add_font.py

```diff
@@ -10,20 +10,13 @@
 HERE = Path(__file__).resolve().parent
 
 
-def test_add_font_non_existing_file():
+def test_add_font_non_existing():
     pdf = FPDF()
-    with pytest.raises(FileNotFoundError) as error:
-        pdf.add_font("non-existing")
-    expected_msg = "[Errno 2] No such file or directory: 'non-existing.pkl'"
-    assert str(error.value) == expected_msg
-
-
-def test_add_font_non_existing_file_unicode():
-    pdf = FPDF()
-    with pytest.raises(FileNotFoundError) as error:
-        pdf.add_font("non-existing", uni=True)
-    expected_msg = "TTF Font file not found: non-existing.pkl"
-    assert str(error.value) == expected_msg
+    for uni in (True, False):
+        with pytest.raises(FileNotFoundError) as error:
+            pdf.add_font("non-existing", uni=uni)
```

Comment:
Does testing for the uni parameter still makes sense when it is deprecated without function?